src/wallet/rpcwallet.cpp

```diff
@@ -2491,6 +2491,8 @@ static UniValue listwallets(const JSONRPCRequest& request)
     return obj;
 }
 
+std::shared_ptr<CWallet> LoadWallet(interfaces::Chain& chain, const WalletLocation& location, std::string& error, std::string& warning);
```

Comment:
It seems a redundant redeclaration. Could be removed.

Reply:
Done.